### DISCUSSION TITLE
Is it possible to set env variable with specific option?

### DISCUSSION THREAD

**Discussion by botzill** [netdata/netdata#12288] [Q&A]
Hi,

I was wondering if I can do smth like `NETDATA_WEB_BIND_TO=*` to set only a specific env.

Thanks.

**Comment by ilyam8** [comment]
Hi, @botzill. No, it is not possible, unfortunately. If you think this would be a good feature, can describe your use case and the feature value proposition - consider creating a [feature request](https://github.com/netdata/netdata/issues/new?assignees=&labels=feature+request%2Cneeds+triage&template=FEAT_REQUEST.yml&title=%5BFeat%5D%3A+).
